docs/README.md

```diff
@@ -0,0 +1,624 @@
+Welcome to the *AIM API - Alpha*!
+
+# Alpha Notice
+
+The AIM API is in an "alpha stage" to gather customer feedback. While in alpha, the API may change in backwards incompatible ways to accommodate for fixes and additions. Breaking changes will be communicated to the primary API contact at least 5 business days in advance.
+
+TSG reserves the right to determine what constitutes a breaking changes. A definition of "breaking changes" will be made available before formal release.
+
+# HTTP API
+
+## API Discovery
+
+In order to reduce client coupling, the AIM API provides an API discovery document available at:
+
+<a class="discovery-config-url"></a>
+
+<pre><code id="discovery-config"></code></pre>
+
+The primary attributes of interest is the `urls` object, which provides static names to full or partial URLs.
```

Comment:
Al:
> Capitalize URL in the previous statement

Reply:
You'll see that `urls` is not just text, but a code/key reference to the code stuff above (it doesn't show here, but does on https://the-strawhecker-group.github.io/aim/). Hence, it needs to stay lowercase/match the code (which has semantic/idiomatic differences).